common/utils.py

```diff
@@ -76,6 +78,17 @@ def cogs_to_agi(self, cogs):
             agi = decimal.Decimal(cogs) * decimal.Decimal(COGS_TO_AGI)
             return agi
 
+    def extract_zip_and_and_tar(self, s3_url):
```

Comment:
Let's have a separate method for extracting and compressing the files.

Reply:
This is addressed